ajax/libs/jquery.turbolinks/package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "jquery.turbolinks",
-  "version": "2.0.2",
+  "version": "2.1.0",
   "filename": "jquery.turbolinks.min.js",
   "author": "Sasha Koss <kossnocorp@gmail.com>",
```

Comment:
Don't we need to keep `Rico Sta. Cruz <hi@ricostacruz.com>` in [package.json](https://github.com/kossnocorp/jquery.turbolinks/blob/master/package.json#L6)? He seems to contribute a lot in this library.